eventkit_cloud/api/views.py

```diff
@@ -399,20 +399,19 @@ def run(self, request, uid=None, *args, **kwargs):
 
     def partial_update(self, request, uid=None, *args, **kwargs):
```

Comment:
Its not clear if the framework handles it, but there should probably be an explicit atomic decorator here.